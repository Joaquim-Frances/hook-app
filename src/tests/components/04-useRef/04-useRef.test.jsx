import { shallow } from "enzyme";
import { RealExampleRef } from "../../../components/04-useRef/RealExampleRef"

describe('pruebas con RealExampleRef', () => {
    
    const wrapper = shallow( <RealExampleRef />);
    
    test('should display correctly', () => {

        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('MultipleCustomHooks').exists()).toBe(false);
        
    })

    test('should display MultipleCustomHooks component', () => {
        
        wrapper.find('button').simulate('click');
        expect(wrapper.find('MultipleCustomHooks').exists()).toEqual(true);
    })
    
    
    
})
