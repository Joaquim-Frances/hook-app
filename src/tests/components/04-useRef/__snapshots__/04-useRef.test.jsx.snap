// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas con RealExampleRef should display correctly 1`] = `
<div>
  <h1>
    Real Example Ref
  </h1>
  <hr />
  <button
    className="btn btn-primary"
    onClick={[Function]}
  >
    Mostrar/Ocultar
  </button>
</div>
`;
